components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/Extensions.kt

```diff
@@ -57,9 +57,12 @@ val Addon.updatedAtDate: Date get() = dateParser.parse(updatedAt)!!
 /**
  * Try to find the default language on the map otherwise defaults to [Addon.DEFAULT_LOCALE].
  */
-internal fun Map<String, String>.translate(addon: Addon): String {
+internal fun Map<String, String>.translate(addon: Addon, context: Context): String {
     val lang = Locale.getDefault().language
-    return get(lang) ?: getValue(addon.defaultLocale)
+    return get(lang) ?: getOrElse(addon.defaultLocale) {
+        // No translation found $lang and not default
```

Comment:
nit: not translation found for $lang and $addon.defaultLoca?

Reply:
The comment wasn't intentional was a leftover of our chat, I will remove it 😄